homeassistant/components/husqvarna_automower/number.py

```diff
@@ -102,3 +119,135 @@ async def async_set_native_value(self, value: float) -> None:
             raise HomeAssistantError(
                 f"Command couldn't be sent to the command queue: {exception}"
             ) from exception
+
+
+@callback
+def _async_get_work_area_cutting_height(data: WorkArea) -> int:
+    """Return the work area cutting height."""
+    if TYPE_CHECKING:
+        # Entity does not get created if it is None
+        assert data is not None
+    return data.cutting_height
+
+
+@callback
+def _async_work_area_mowers(data: dict[int, WorkArea] | None) -> dict[int, WorkArea]:
+    """Return the work areas of the mower."""
+    if TYPE_CHECKING:
+        # Entity does not get created if it is None
+        assert data is not None
+    return data
```

Comment:
I think this should not be needed, since it's only type checking, but I'm not sure if I can directly see how

Reply:
done in https://github.com/home-assistant/core/pull/116115/commits/2d4b282a22f3257f0bd5677094ca5d3a28367390